demisto_sdk/commands/lint/tests/test_pylint_plugin/xsoar_checker_test.py

```diff
@@ -0,0 +1,137 @@
+import astroid
+import pylint.testutils
+from demisto_sdk.commands.lint.resources.pylint_plugins import base_checker
```

Comment:
do you want to create a single file for all pylint? or different ones? please re name the file accordingly anyhow.

Reply:
There will be a test file for each plugin when they will be implemented.